libs/model/src/models/reaction.ts

```diff
@@ -1,12 +1,16 @@
 import { EventNames, logger, stats } from '@hicommonwealth/core';
-import Sequelize from 'sequelize';
+import Sequelize, { CreateOptions } from 'sequelize';
 import { emitEvent, getThreadContestManagers } from '../utils';
 import type { AddressAttributes } from './address';
 import type { CommunityAttributes } from './community';
 import type { ModelInstance } from './types';
 
 const log = logger(import.meta);
 
+type EnrichedReactionCreateOptions = CreateOptions<ReactionAttributes> & {
```

Comment:
is this to not emit events? you can encapsulate this logic with an ENV var that get's checked inside emitEvent?

Reply:
I'm actually removing that. Not needed anything since bulk create doesn't trigger the hooks